src/components/vm/nics/nicBody.tsx

```diff
@@ -271,3 +302,355 @@ export const NetworkTypeAndSourceRow = ({
         </>
     );
 };
+
+export function portForwardText(pf: VMInterfacePortForward): string {
+    let text = "";
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2331-c59dd4a9-20251031-071718-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)